src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php

```diff
@@ -0,0 +1,111 @@
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product\Image;
+
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\Form\Util\StringUtil;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * This form type integrates a Dropzone used to manage images, the twig templates is a simple div
+ * with data attributes, it needs javascript from the Dropzone.vue component to work.
+ *
+ * This form type though should be used to pass any configuration to the javascript components, like
+ * translations, locales, or productId (maybe even urls could be configurable).
+ */
+class ImageDropzoneType extends TranslatorAwareType
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        parent::buildForm($builder, $options);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+        $resolver->setDefaults([
+            'translations' => [
+                'window.selectAll' => $this->trans('Select all', 'Admin.Actions'),
+                'window.settingsUpdated' => $this->trans('Settings udpated', 'Admin.Actions'),
+                'window.imageReplaced' => $this->trans('Image replaced', 'Admin.Actions'),
+                'window.unselectAll' => $this->trans('Unselect all', 'Admin.Actions'),
+                'window.replaceSelection' => $this->trans('Replace selection', 'Admin.Actions'),
+                'window.cantDisableCover' => $this->trans('You cannot disable the cover image.', 'Admin.Actions'),
+                'window.selectedFiles' => $this->trans(
+                    '[1]%filesNb%[/1] selected file(s)',
+                    'Admin.Actions',
+                    ['[1]' => '<span>', '[/1]' => '</span>']
+                ),
+                'window.useAsCover' => $this->trans('Use as cover image', 'Admin.Actions'),
+                'window.saveImage' => $this->trans('Save image settings', 'Admin.Actions'),
+                'window.delete' => $this->trans('Delete selection', 'Admin.Actions'),
+                'window.close' => $this->trans('Close window', 'Admin.Actions'),
+                'window.zoom' => $this->trans('Zoom on selection', 'Admin.Actions'),
+                'window.fileisTooLarge' => $this->trans(
+                    'The file is too large. Maximum size allowed is => [1] MB. The file you are trying to upload is [2] MB.',
```

Comment:
'The file is too large. The maximum size allowed is [1] MB. The file you are trying to upload is [2] MB.'

I don't think we need the '=>' here.